src/main/java/Duke.java

```diff
@@ -20,33 +20,35 @@ private static void printHorizontalLine() {
         System.out.println("____________________________________________________________");
     }
 
-    public static void main(String[] args) {
+    private static void printLogo() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+    }
 
```

Comment:
I like how you changed the functions to private

Reply:
Thank you, I put it as private in line with the encapsulation principle!